src/orb.yml.hbs

```diff
@@ -530,24 +535,28 @@ commands:
           "The deployment controller to use for the service. Defaulted to ECS"
         type: string
```

Comment:
I missed out on this in my earlier review. Could this be changed to an enum too? 🙂

Reply:
Sorry, I missed too. Fixed by 500ac8c .